wm8960.c

```diff
@@ -1383,7 +1383,11 @@ static const struct snd_soc_component_driver soc_component_dev_wm8960 = {
 	.idle_bias_on		= 1,
 	.use_pmdown_time	= 1,
 	.endianness		= 1,
+#if LINUX_VERSION_CODE < KERNEL_VERSION(6,0,0)
 	.non_legacy_dai_naming	= 1,
+#else
+	.legacy_dai_naming	= 0,
```

Comment:
Do we need these two lines? Could you please remove them?

Reply:
Not sure they are needed, but they seemed logical, given the filed name (meaning?) change from **.non_legacy_dai_naming** to **.legacy_dai_naming**.

Reply:
The field is not explicitely set to 0 upstream, so we'd better remove it to avoid diverging too much.

Reply:
Done...